sdk/resourcemanager/Azure.ResourceManager.Core/tests/Unit/IdentityTests.cs

```diff
@@ -182,17 +176,6 @@ public void TestDeserializerValidSystemAndUserAssigned()
             Assert.AreEqual("77563a98-c9d9-407b-a7af-592d21fa2153", user.Values.First().PrincipalId.ToString());
         }
 
-        [TestCase]
-        public void TestDeserializerInvalidType()
```

Comment:
why are we getting rid of the invalid deserialization tests?

Reply:
Reverted back.